pkg/resource/table/hooks.go

```diff
@@ -133,7 +134,9 @@ func isTableUpdating(r *resource) bool {
 		return false
 	}
 	dbis := *r.ko.Status.TableStatus
-	return dbis == string(v1alpha1.TableStatus_SDK_UPDATING)
+	return dbis == string(v1alpha1.TableStatus_SDK_UPDATING) ||
+		*r.ko.Spec.ContributorInsights == string(svcsdktypes.ContributorInsightsStatusEnabling) ||
+		*r.ko.Spec.ContributorInsights == string(svcsdktypes.ContributorInsightsStatusDisabling)
```

Comment:
I think this deserves to be in a seperate function `isTableContributorInsightsUpdating` and you can call `if isTableUpdating(...) && isTableContributorInsightsUpdating(...)`